dll/shellext/zipfldr/CZipEnumerator.hpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * PROJECT:     ReactOS Zip Shell Extension
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     CZipEnumerator
+ * COPYRIGHT:   Copyright 2017 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+struct CZipEnumerator
+{
+private:
+    CComPtr<IZip> spZip;
+    bool first;
+    CAtlList<CStringA> _returned;
```

Comment:
You don't use the usual "m_Blah;" notation for class members here?

Reply:
I will rename those to be consistent.